### PR TITLE
Adding loader for Four-Way Tabla dataset (ISMIR 21)

### PR DESCRIPTION
**Pull request by genisplaja**
### Description

Please include the following information at the top level docstring for the dataset's module mydataset.py:

- [x] Describe annotations included in the dataset
- [x] Indicate the size of the datasets (e.g. number files and duration, hours)
- [x] Mention the origin of the dataset (e.g. creator, institution)
- [x] Describe the type of music included in the dataset
- [x] Indicate any relevant papers related to the dataset
- [x] Include a description about how the data can be accessed and the license it uses (if applicable)

#### Dataset loaders checklist:

- [x] Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file.
- [x] Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`.
- [x] Create a module in mirdata, e.g. `mirdata/my_dataset.py`
- [x] Create tests for your loader in `tests/datasets/`, e.g. `test_my_dataset.py`
- [x] Add your module to `docs/source/mirdata.rst` and `docs/source/quick_reference.rst`
- [x] Run `tests/test_full_dataset.py` on your dataset.

### CODE REVIEW

**Comment by genisplaja** [comment]
The loader is ready :) All tests are passing on my machine. Feel free to review @nkundiushuti.

File: mirdata/datasets/four_way_tabla.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            version,
+            name="four_way_tabla",
+            track_class=Track,
+            bibtex=BIBTEX,
+            indexes=INDEXES,
+            remotes=REMOTES,
+            license_info=LICENSE_INFO,
+        )
+
+    def load_audio(self, *args, **kwargs):
```
Comment (nkundiushuti) [inline mirdata/datasets/four_way_tabla.py:237]: this looks like it was not covered by tests. can you do a test_load_audio as here:
https://github.com/mir-dataset-loaders/mirdata/blob/master/tests/datasets/test_cante100.py

File: mirdata/datasets/four_way_tabla.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+        return load_audio(*args, **kwargs)
+
+    def load_onsets(self, *args, **kwargs):
```
Comment (nkundiushuti) [inline mirdata/datasets/four_way_tabla.py:240]: weirdly this also looks not covered by tests although I see that you do test load_onsets in test_get_onsets

**Review by nkundiushuti — CHANGES_REQUESTED** [review]
it looks good, maybe I would write one more test to increase test coverage.

**Comment by genisplaja** [comment]
Hey @nkundiushuti :) I just removed these definitions here, everything looks like it's working fine, and the coverage problem has been solved. I guess the loader would be ready now!

**Review by harshpalan — APPROVED** [review]
Hello @genisplaja, thanks for contributing to mirdata. I'm approving this request. @nkundiushuti Please go ahead with the review and merge. Thanks :)

**Review by nkundiushuti — APPROVED** [review]

**Pull request merged**